BrainPortal/app/models/demand.rb

```diff
@@ -97,15 +72,15 @@ def approved? #:nodoc:
     self.approved_by.present? && self.approved_at.present?
   end
 
-  def is_suspicious?  # types: 1=warning, 2=weird_entries, 3=keyboard_banging
+  def is_suspicious? #:nodoc:
```

Comment:
I thought that methods ending with a question mark should return `true` or `false`. I'm not sure how useful the suspiciousness feature is anyway.